common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -504,6 +523,46 @@ def metadata_cascades_for_dep(dependency)
         ).to_s
       end
 
+      def ignore_conditions_table
+        # Return an empty string if ignore_conditions is empty
+        return "" if @ignore_conditions.empty?
+
+        # Filter out the conditions where from_config_file is false and dependency is in @dependencies
+        valid_ignore_conditions = @ignore_conditions.select do |ic|
+          !ic[:from_config_file] && dependencies.any? { |dep| dep.name == ic[:dependency_name] }
```

Comment:
@honeyankit where does this `from_config_file` value come from? I don't see it in https://github.com/github/dependabot-api/blob/9fc5f8ad58c303bb9a1df9e460ee812d5350e797/app/models/update_job.rb#L184-L187

Reply:
@jurre: We also need to modify this function. My earlier assumption was wrong about the attributes of ignore_conditions, We have to replace `from_config_file` with `source`

Reply:
I had implemented `@dependbot show <dep -name> ignore conditions` and implement something [similar](https://github.com/github/dependabot-api/pull/4311/files#diff-c2b744d74da5d1462a2bd142ab16ff464fa3ed91f0310f248eb650f5db506245R42-R75). We have to change the `ignore_conditions_table` method entirely.

Reply:
I have added the field `created_at` and `updated_at` in update job on this PR: https://github.com/github/dependabot-api/pull/4319